pycytominer/cyto_utils/cells.py

```diff
@@ -846,3 +775,67 @@ def aggregate_profiles(
             )
         else:
             return aggregated
+
+
+def _sqlite_strata_conditions(df, dtypes, n=1):
+    """Given a dataframe where columns are merge_cols and rows are unique
+    value combinations that appear as aggregation strata, return a list
+    of strings which constitute valid SQLite conditional statements.
+
+    Parameters
+    ----------
+    df : pandas.core.frame.DataFrame
+        A dataframe where columns are merge_cols and rows represent
+        unique aggregation strata of the compartment table
```

Comment:
```suggestion
        the specific image IDs grouped by unique aggregation strata (e.g. Metadata_well) of the image table
```

Is this more accurate?

Reply:
Yes that's right.  Only I think it turns out to be whatever is in `merge_cols`, so "ImageNumber" and "TableNumber"